src/dataframe/dataframe.jl

```diff
@@ -1179,6 +1185,8 @@ to vertically concatenate data frames.
 
 # Examples
 ```jldoctest
+julia> using DataFrames
```

Comment:
Why is this line needed if you added https://github.com/JuliaData/DataFrames.jl/pull/2703/files#diff-4aae2d1c783cade58bd2cb13748da956e568b7f2aed5fafd9e2a46fb97daf613R2?

Reply:
Yeah, I have removed that line in my latest commit.